core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/services/restart/ClusterServicesRestartActions.java

```diff
@@ -49,21 +46,11 @@ protected Selectable createRequest(ClusterViewContext context) {
         };
     }
 
-    @Bean(name = "CLUSTER_SERVICE_RESTART_POLLING_STATE")
-    public Action<?, ?> clusterStartPolling() {
-        return new AbstractClusterAction<>(ClusterServicesRestartResult.class) {
-            @Override
-            protected void doExecute(ClusterViewContext context, ClusterServicesRestartResult payload, Map<Object, Object> variables) {
-                sendEvent(context, new ClusterServicesRestartPollingRequest(context.getStackId(), payload.getRequestId()));
-            }
-        };
-    }
-
```

Comment:
Is it safe to delete this step from the flow? What would happen if the flow is in CLUSTER_SERVICE_RESTART_FINISHED_STATE state when CB is restarted and the flow engine fires the ClusterServicesRestartPollingResponse event again to restart the flow?

Reply:
I tested this case and it will continue the flow with the CLUSTER_START_FINISHED_STATE